docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md

```diff
@@ -0,0 +1,559 @@
+
+ElasticSearch will use S3 as a backup and restore bucket and for that, you need to configure a Kubernetes secret to not expose those in cleartext.
+
+You can pull the data from Terraform since you exposed those via the `output.tf`.
+
+1. Where your `config.tf` and `output.tf` live, execute the following to export the access keys to environment variables. This will allow an easier creation of the Kubernetes secret via the command line.
+
+```bash
+export ACCESS_KEY=$(terraform output -raw s3_aws_access_key_id)
+export SECRET_ACCESS_KEY=$(terraform output -raw s3_aws_secret_access_key)
+```
+
+2. Create the Kubernetes secret within the `camunda-london` and `camunda-paris` namespace on each respective cluster.
+
+```bash
+kubectl --context $CLUSTER_1 -n $CAMUNDA_NAMESPACE_1 create secret generic elasticsearch-env-secret \
+    --from-literal=S3_ACCESS_KEY=$ACCESS_KEY \
+    --from-literal=S3_SECRET_KEY=$SECRET_ACCESS_KEY
+
+kubectl --context $CLUSTER_2 -n $CAMUNDA_NAMESPACE_2 create secret generic elasticsearch-env-secret \
+    --from-literal=S3_ACCESS_KEY=$ACCESS_KEY \
+    --from-literal=S3_SECRET_KEY=$SECRET_ACCESS_KEY
+```
+
+</TabItem>
+<TabItem value="helm-chart" label="Camunda 8 Helm Chart prerequisites">
+
+Our recommendation is to work with layered helm values files since you will want to have a base `value.yml` that is generally applicable and then two overlays that are for `region0` and `region1`.
+
+#### Base values.yml
+
+The following is an example `values.yml` that forms the base and ca be further extended.
+
+Especially important within this is the `global.multiregion.regions` directive to be two for two regions. Additionally, to disable identity and optimize since those are currently not working within the setup.
+
+You need to adjust the `placeholder` for `ZEEBE_BROKER_CLUSTER_INITIALCONTACTPOINTS` to reflect your current setup. These are the contact points for the brokers to know how to form the cluster. More information on what the variable means in the [setting up a cluster documentation](../../../../zeebe-deployment/operations/setting-up-a-cluster.md).
+
+In our example, the `placeholder` would is being replaced with:
+
+```yaml
+-name: ZEEBE_BROKER_CLUSTER_INITIALCONTACTPOINTS
+ value: "camunda-zeebe-0.camunda-zeebe.camunda-london.svc.cluster.local:26502,camunda-zeebe-1.camunda-zeebe.camunda-london.svc.cluster.local:26502,camunda-zeebe-2.camunda-zeebe.camunda-london.svc.cluster.local:26502,camunda-zeebe-3.camunda-zeebe.camunda-london.svc.cluster.local:26502,camunda-zeebe-0.camunda-zeebe.camunda-paris.svc.cluster.local:26502,camunda-zeebe-1.camunda-zeebe.camunda-paris.svc.cluster.local:26502,camunda-zeebe-2.camunda-zeebe.camunda-paris.svc.cluster.local:26502,camunda-zeebe-3.camunda-zeebe.camunda-paris.svc.cluster.local:26502"
+```
+
+The zeebe cluster size is 8 with a dual-region setup, resulting in 4 brokers per region.
+
+The address forms as follows:
+
+```bash
+$pod.$service.$namespace.svc.cluster.local:26502
```

Comment:
can something like https://github.com/camunda-community-hub/camunda-8-helm-profiles/blob/main/google/multi-region/active-active/setup-zeebe.py be used as a endpoint hostnames list generator?

i.e. a script put in https://github.com/camunda/c8-multi-region/tree/aws-operational/test/resources/aws/2-region/terraform and asking some basic inputs from a user / env vars (namespace name, broker count ...). It's a bit hard to generate those by hand without mistakes.

Reply:
yes, we can probably have a small python script.

Reply:
a Python script to run would be yet another thing easier with the `git clone` approach should we go for it.

Reply:
was handled in - https://github.com/camunda/c8-multi-region/pull/17